core/src/trezor/ui/__init__.py

```diff
@@ -78,7 +81,8 @@ def pulse(period: int, offset: int = 0) -> float:
     return 0.5 + 0.5 * math.sin(2 * math.pi * (utime.ticks_us() + offset) / period)
 
 
-async def alert(count: int = 3) -> None:
+async def _alert(count: int) -> None:
+    global _alert_in_progress
```

Comment:
maybe move this line to the end of the function, just before the assignment?

Reply:
Done. Rebased on top of master.